darts/tests/models/forecasting/test_torch_forecasting_model.py

```diff
@@ -291,6 +291,145 @@ def test_manual_save_and_load(self, tmpdir_fn):
         # compare chained load_from_checkpoint() save() with manual save
         assert model_chained_load_save.predict(n=4) == model_manual_save.predict(n=4)
 
+    def test_manual_save_and_load_with_drop_training(self, tmpdir_fn):
```

Comment:
This test is almost a copy of the `test_manual_save_and_load` test.
Could you try to extend that test to also check for the clean safe?

And then as a last point it would be good to also check that the callbacks were removed.